hello-key/config.ml

```diff
@@ -0,0 +1,14 @@
+open Mirage
+
+let key =
+  let doc = Key.Arg.info ~doc:"How to say hello." ["hello"] in
+  Key.(create "hello" Arg.(opt string "Hello World!" doc))
+
+let main =
+  foreign
+    ~keys:[Key.abstract key]
```

Comment:
I'm fine with `Key.v` being shorthand for `Key.abstract` in Functoria/Mirage generally, but I wouldn't want to put it in example code without a link to a nice document explaining what it means and why it's named that way.  This kind of naming is a big stumbling block for me when trying to understand new codebases because it's both hard to remember and hard to search for.